examples/components/toast/toast.styles.scss

```diff
@@ -0,0 +1,17 @@
+@import '~styles/theme';
+.example {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    padding: 1em;
+}
+
+.button{
+    background: $secondary-color;
+    color: #ffffff;
+    padding: 1em;
+    margin: 1em;
+    width: 50%;
+    cursor: pointer;
```

Comment:
not the right order, sass-lint where are you!?
@guillaumecrespel have we forgotten the sass-linter ?

Reply:
//sass-lint.yml == >   include: 'src/**/*.scss'   :)